convert2rhel.ini.example

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Example of configuration file convert2rhel.ini for secrets.
+# Possible locations of this file:
+# 1) user specified and passed by -c, --config-file option; highest priority
+# 2) ~/convert2rhel.ini; lower priority
+# 3) /etc/convert2rhel.ini; the lowest priority
+
+# [SECRET]
```

Comment:
Since the config file can later be used to not set only secrets but also to set various other things, I'd keep using the header there.

And because there might be more secrets later on, for instance network proxy settings, we should name the header more specifically so that it's clear what exactly the keys relate to.
```suggestion
[subscription-manager]
```
Also, headers don't need to be commented out I'd say.